chromium/browser_type.go

```diff
@@ -240,23 +244,27 @@ func (b *BrowserType) allocate(
 		return nil, err
 	}
 
-	path := opts.ExecutablePath
-	if path == "" {
-		path = b.ExecutablePath()
-	}
-
 	return common.NewLocalBrowserProcess(bProcCtx, path, args, dataDir, bProcCtxCancel, logger) //nolint: wrapcheck
 }
 
-// ExecutablePath returns the path where the extension expects to find the browser executable.
-func (b *BrowserType) ExecutablePath() (execPath string) {
-	if b.execPath != "" {
-		return b.execPath
+// ErrChromeNotInstalled is returned when the Chrome executable is not found.
+var ErrChromeNotInstalled = errors.New("chrome or chromium is not installed on this system")
```

Comment:
Strictly speaking this message seems to say that one or the other is not installed and therefore there's an error. Should it be something like:
```suggestion
var ErrChromeNotInstalled = errors.New("neither chrome nor chromium are installed in this system")
```

Reply:
Ah, right, thanks 👍 Should we say "is": neither chrome nor chromium is installed on this system"?

Reply:
I would think "are" is OK, as we are speaking about two different subjects, but maybe I'm wrong :thinking:

Reply:
@heitortsergent WDYT? :)

Reply:
I'm wondering since the linked article suggests: _When neither and nor link singular terms, the verb is singular: Neither the car nor the truck is available._

Reply:
@ka3de See: bbfea56f.